nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -74,6 +80,20 @@ let
 
   drivesCmdLine = drives: concatStringsSep " " (imap1 driveCmdline drives);
 
+  driveDeviceName = idx:
+    let letter = elemAt lowerChars idx;
+    in if cfg.qemu.diskInterface == "scsi" then
+      "/dev/sd${letter}"
+    else
+      "/dev/vd${letter}";
+
+  lookupDriveDeviceName = driveName: driveList:
+    (findSingle (drive: drive.name == driveName)
+    (throw "Drive ${driveName} not found")
+    (throw "Multilple drives named ${driveName}") driveList).device;
```

Comment:
```suggestion
    (throw "Multiple drives named ${driveName}") driveList).device;
```

Reply:
Addressed in the PR that adds this: https://github.com/NixOS/nixpkgs/pull/92205#discussion_r449770139